src/pages/task/TestsTable.tsx

```diff
@@ -10,11 +14,27 @@ import {
 } from "types/task";
 import { Categories } from "gql/queries/get-task-tests";
 import queryString from "query-string";
+import styled from "@emotion/styled";
 
 enum DefaultQueryParams {
   Sort = "1",
   Category = "TEST_NAME"
 }
+const arrayFormat = "comma";
+const updateTestNameQueryParam = debounce(
+  (
+    testName: string,
+    search: string,
+    replace: (path: string) => void,
+    pathname: string
+  ) => {
+    const parsed = queryString.parse(search, { arrayFormat });
+    parsed[RequiredQueryParams.TestName] = testName;
+    const nextQueryParams = queryString.stringify(parsed);
+    replace(`${pathname}?${nextQueryParams}`);
+  },
+  500
```

Comment:
do you think 500 is too long? we do want it to feel instantaneous without actually being instantaneous. maybe 200-300?

Reply:
500 is a little noticeable, i changed it to 250.